tests/conftest.py

```diff
@@ -1,5 +1,10 @@
 import pytest
 import os
+
+# suppress all logging unless location defined in ENV
+if os.getenv("GARAK_LOG_FILE", None) is None:
+    os.environ["GARAK_LOG_FILE"] = str(os.devnull)
```

Comment:
we, uh, we do no tests of the logging system?

Reply:
This is added because now any import of a `garak` module will trigger logging to start. While we have no test that validates the logging. As of this PR, if we do not suppress using `GARAK_LOG_FILE`, logging calls will generate data in the default `garak.log` during test execution.

By suppressing here only if the ENV var is not set we enable developer testing to generate logs during debugging if desired.